tools/envoy_headersplit/replace_includes_test.py

```diff
@@ -0,0 +1,130 @@
+# Lint as: python3
+"""Tests for headersplit."""
+
+import unittest
+from unittest import mock
+import os
+from pathlib import Path
+import replace_includes
+
+
+class ReplaceIncludesTest(unittest.TestCase):
+
+  def test_to_classname(self):
+    # Test file name with whole path
+    self.assertEqual(replace_includes.to_classname("test/mocks/server/admin_stream.h"),
+                     "MockAdminStream")
+
+    # Test file name without .h extension
+    self.assertEqual(replace_includes.to_classname("cluster_mock_priority_set"),
+                     "MockClusterMockPrioritySet")
+
+  def test_to_bazelname(self):
+    # Test file name with whole path
+    self.assertEqual(replace_includes.to_bazelname("test/mocks/server/admin_stream.h", "server"),
+                     "//test/mocks/server:admin_stream_mocks")
+
+    # Test file name without .h extension
+    self.assertEqual(replace_includes.to_bazelname("cluster_mock_priority_set", "upstream"),
+                     "//test/mocks/upstream:cluster_mock_priority_set_mocks")
+
+  class Fake_dir():
```

Comment:
Per the style guide, class names should be in `PascalCase`, e.g. `FakeDir`.

Reply:
fixed